proof/crefine/RISCV64/Invoke_C.thy

```diff
@@ -2413,7 +2362,7 @@ lemma invokeUntyped_Retype_ccorres:
            apply (rule sz_bound[folded maxUntypedSizeBits_def])
           apply clarsimp
           apply (wp updateFreeIndex_forward_invs' sch_act_simple_lift
-                    updateFreeIndex_cte_wp_at hoare_vcg_const_Ball_lift
+                    Untyped_R.updateFreeIndex_cte_wp_at hoare_vcg_const_Ball_lift
```

Comment:
This qualification being needed made me check and it looks like we should be able to remove the lemma in `Untyped_R` and use the `Finalise_R` version. It's definitely not worth cleaning up now, but we might want to leave a FIXME.

Reply:
The qualification was actually easy to remove after all, so I have just done that